examples/bayesian_regression.py

```diff
@@ -19,11 +19,13 @@
 
 
 # generate toy dataset
-def build_linear_dataset(N, p, noise_std=0.1):
-    X = np.linspace(-6, 6, num=N)
-    y = 3 * X + 1 + np.random.normal(0, noise_std, size=N)
-    X = X.reshape((N, p))
-    y = y.reshape((N, 1))
+def build_linear_dataset(N, p, noise_std=0.01):
+    X = np.random.rand(N, p)
+    # use random weights from [0, 8]
```

Comment:
nit: `np.random.randint(0, 8)` actually draws from `[0,8) = [0,7]`.

Reply:
makes sense, i picked a random int. ill change the comment